dynamic-config.md

```diff
@@ -173,54 +140,166 @@ For detailed parameter description, refer to [PD Configuration File](/pd-configu
 | `{db-name}.{cf-name}.level0-slowdown-writes-trigger` | The maximum number of files at L0 that trigger write stall |
-| `{db-name}.{cf-name}.level0-stop-writes-trigger` | The maximum number of files at L0 required to completely block write |
+| `{db-name}.{cf-name}.level0-stop-writes-trigger` | The maximum number of files at L0 that completely block write |
 | `{db-name}.{cf-name}.max-compaction-bytes` | The maximum number of bytes written into disk per compaction |
 | `{db-name}.{cf-name}.max-bytes-for-level-multiplier` | The default amplification multiple for each layer |
 | `{db-name}.{cf-name}.disable-auto-compactions` | Enables or disables automatic compaction |
 | `{db-name}.{cf-name}.soft-pending-compaction-bytes-limit` | The soft limit on the pending compaction bytes |
 | `{db-name}.{cf-name}.hard-pending-compaction-bytes-limit` | The hard limit on the pending compaction bytes |
 | `{db-name}.{cf-name}.titan.blob-run-mode` | The mode of processing blob files |
+| `storage.block-cache.capacity` | The size of shared block cache (supported since v4.0.3) |
+| `backup.num-threads` | The number of backup threads (supported since v4.0.3) |
+| `split.qps-threshold` | The threshold to execute `load-base-split` on a Region. If the read QPS is higher than this value for 10 consecutive seconds, this Region should be split. |
+| `split.split-balance-score` | The parameter of `load-base-split`, which ensures the load of the two split Regions is as balanced as possible |
+| `split.split-contained-score` | The parameter of `load-base-split`, which reduces the cross-Region visits after split as much as possible |
 
 In the table above, parameters with the `{db-name}` or `{db-name}.{cf-name}` prefix are configurations related to RocksDB. The optional values of `db-name` are `rocksdb` and `raftdb`.
 
 - When `db-name` is `rocksdb`, the optional values of `cf-name` are `defaultcf`, `writecf`, `lockcf`, and `raftcf`.
 - When `db-name` is `raftdb`, the value of `cf-name` can be `defaultcf`.
 
 For detailed parameter description, refer to [TiKV Configuration File](/tikv-configuration-file.md).
+
+### Modify PD configuration online
+
+Currently, PD does not support the separate configuration for each instance. All PD instances share the same configuration.
+
+You can modify the PD configurations using the following statement:
+
+{{< copyable "sql" >}}
+
+```sql
+set config pd log.level="info"
+```
+
+If the modification is successful, `Query OK` is returned:
+
+```sql
+Query OK, 0 rows affected (0.01 sec)
+```
+
+If a configuration item is successfully modified, the result is persisted in etcd instead of the configuration file; the configuration in etcd will prevail in the subsequent operations. The names of some configuration items might conflict with TiDB reserved words. For these configuration items, use backtick `` ` `` to enclose them. For example, `` `schedule.leader-schedule-limit` ``.
```

Comment:
```suggestion
If a configuration item is successfully modified, the result is persisted in etcd instead of in the configuration file; the configuration in etcd will prevail in the subsequent operations. The names of some configuration items might conflict with TiDB reserved words. For these configuration items, use backtick `` ` `` to enclose them. For example, `` `schedule.leader-schedule-limit` ``.
```